interface/DrawFunctions.C

```diff
@@ -11,7 +11,7 @@ TH2F* NewOverQCD = 0;
 
 bool Under(float phi, float eta) {
   if (ECALmap == 0) {
-    ECALmap = TFile::Open("/scratch/sdonato/VHbbRun2/V21/CMSSW_7_1_5/src/Xbb/python/plot.root", "r");
+    ECALmap = TFile::Open("/afs/cern.ch/work/s/swang373/private/sync_V21/src/Xbb/python/plot.root", "r");
```

Comment:
The root file it refers to is generated by getPeaks2.py in the ZvvHbb13TeVMacros directory, so as long as the macro outputs it to the same directory within Xbb, a relative path should work.